### ISSUE TITLE
sdrs_protectiongroup_v1 please clarify domain_id

### ISSUE THREAD

**Issue by rramge** [opentelekomcloud/terraform-provider-opentelekomcloud#1930]
Hello,

The documentation states:
[domain_id](https://registry.terraform.io/providers/opentelekomcloud/opentelekomcloud/latest/docs/resources/sdrs_protectiongroup_v1#domain_id) - (Required) Specifies the ID of an active-active domain.

The given example is not helpful:

```
[...]
domain_id     = var.domain_id
[...]
```

There is a datasource, sdrs_domain_v1, but this one requires the name of an already existing domain:

[name](https://registry.terraform.io/providers/opentelekomcloud/opentelekomcloud/latest/docs/data-sources/sdrs_domain_v1#name) - (Optional) Specifies the name of an active-active domain.

A datasource which lists the available names is missing. My first guess is that it might be the OTC domain name, "OTC-EU-DE-000000000010000xxxxx", but this leads to an error message:

```
╷
│ Error: your query returned no results. Please change your filters and try again.
│ 
│   with data.opentelekomcloud_sdrs_domain_v1.domain-001,
│   on 12-sdrs.tf line 41, in data "opentelekomcloud_sdrs_domain_v1" "domain-001":
│   41: data "opentelekomcloud_sdrs_domain_v1" "domain-001" {
│ 
```

So it must be something else. 

I tried the OTC webconsole, but during creation there is no required or optional field for domain_id in the UI, so it seems to be an internal thing and not necessarily a required input from the user.

Please clarify and update the documentation accordingly.

**Comment by artem-lifshits** [comment]
Hello @rramge ``active-active domain`` can be checked via ``data-source/sdrs_domain_v1``. Resource ID will be the one for ``active-active domain``.

 I will fix documentation with additional info & examples.

**Comment by rramge** [comment]
@artem-lifshits Yes, I saw the datasource, but it didn't help. 
If one already exists, how can I find out the name?
If none already exists, how can I create it?

**Comment by artem-lifshits** [comment]
@rramge at the moment there's only one ``active-active domain`` available with no way to create additional one.
``Name`` parameter can be specified in case OTC supports more ``active-active domains`` in the future.

So I suggest using data source without specifying any name:
`data "opentelekomcloud_sdrs_domain_v1" "dom_1" {}`

**Comment by artem-lifshits** [comment]
@rramge documentation was fixed in the latest release ``v1.31.3``.
